ios/manifest

```diff
@@ -2,18 +2,18 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 3.0.0
+version: 3.0.1
 apiversion: 2
 architectures: arm64 x86_64
 description: ti.animation
 author: Hans Knöchel
 license: Apache 2
-mac: false
+mac:true 
 copyright: Copyright (c) 2017-present by Hans Knöchel, Michael Gangolf
 
 # these should not be edited
 name: ti.animation
 moduleid: ti.animation
 guid: 1fcfea69-bbbf-42f8-b738-9545b5dda74d
 platform: iphone
-minsdk: 9.2.0
+minsdk: 10.0.0 
```

Comment:
9.2.0 should be fine for macOS Catalyst I think :)

Reply:
my tests are leaded to the resolution min. iOS 12.1 also TitaniumKit.xcframwork was needed to recompile to fullfil the requirements... As far as I know, also iOS 13 will be needed when Xcode 13 will be released.... (M1 Mac tested)

Reply:
TitaniumKit: definitely. But I can't see why 12.1 would be a Lottie requirement. Their own target is actually iOS 9+. And Xcode 13 will not raise the minimum (you could even target a minimum of iOS 7.x if you want to).

Reply:
right, but I was not able to build,  maybe because my app I'm currently coding uses CoreML etc.... (I know iOS 11).... at all I heavily used GPUImage-Module... now for Catalyst not longer usable because of OpenGL (https://developer.apple.com/forums/thread/126450) ... however I encountered Xcode build errors everytime when the sources (PODS etc....) are set to lower then iOS 12....  Hey, this is my first PR ever! :-D